### DISCUSSION TITLE
Is there a way to set up automatic follow up emails based on conditions

### DISCUSSION THREAD

**Discussion by heisenberg2980** [forwardemail/forwardemail.net#263] [General]
I would like to know if there is a way to set up automatic follow-up emails based on conditions, something similar to this tool that exists for Office: https://ivasoft.com/ifrepliedflow.shtml

I am using Thunderbird to access the email, in case that makes any difference.

**Comment by titanism** [comment]
The best thing you could do is set up a webhook, and then programmatically trigger a follow-up email using the webhook and our outbound SMTP service (or use our API to send an email over HTTP).

https://forwardemail.net/faq#do-you-support-webhooks
https://forwardemail.net/api
